.github/workflows/kitchen.yml

```diff
@@ -124,3 +122,27 @@ jobs:
         working-directory: kitchen-tests
         run:  |
           bundle exec kitchen test end-to-end-${{ matrix.os }}
+  linux-2004-host:
```

Comment:
Can we add a comment here stating we have to use 20.04 host for these operating other wise the dokken images throw timedatectl error?

Reply:
Done.